### PR TITLE
added document histories and fixed version number for interop spec

### PR DESCRIPTION
**Pull request by tulshi**

### CODE REVIEW

File: openid-sharedsignals-framework-1_0.md [hunk 1]
```diff
@@ -2278,6 +2278,22 @@ The technology described in this specification was made available from contribut
...
+    * Fix description of error code for invalid state
+    * Add SHOULD language about checking the issuer value
+    * Added language requiring authorization of stream management API
+    * Added description of `txn` claim
+    * Added a `default_subjects` field to Transmitter Configuration Metadata indicating expected subject behavior for new streams
+    * Editorial: Standardize terms and casing, fix some typos
+
   -02
 
     * added txn claims to non-normative SET examples and generic txn callout under SET Profile section RFC8417(#152)
```
Comment (appsdesh) [inline openid-sharedsignals-framework-1_0.md:2299]: This should move to `- 03`

**Review by appsdesh — CHANGES_REQUESTED** [review]

File: openid-sharedsignals-framework-1_0.md (hunk 1 above)
Comment (tulshi) [reply openid-sharedsignals-framework-1_0.md:2299]: That section shows the history of what changed between 01 and 02. The section for changes between 02 and 03 is above this.

Comment (appsdesh) [reply openid-sharedsignals-framework-1_0.md:2299]: > added txn claims to non-normative SET examples and generic txn callout under SET Profile section RFC8417(#152)

This came in after 02 hence should be added to 03 list

Comment (tulshi) [reply openid-sharedsignals-framework-1_0.md]: fixed, I didn't realize that PR had added to the document history section.

**Review by appsdesh — APPROVED** [review]
LGTM!

**Review by iamseanodentity — APPROVED** [review]

**Review by timcappalli — COMMENTED** [review]
LGTM

**Pull request merged**